knexfile.js

```diff
@@ -0,0 +1,18 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/lambda.db3'
+    },
+    migrations: {
+      directory: './migrations'
```

Comment:
you may want to add in the /data/ folder path in front of the migration and seed directories to keep your files more organized